features/step_definitions/project_member_steps.rb

```diff
@@ -84,24 +84,12 @@ def select_principal(principal)
 
 def select_role(role)
   if !User.current.impaired?
-    select_within_select2(role.name, '#s2id_member_role_ids')
+    select2(role.name, css: '#s2id_member_role_ids')
   else
     select_without_select2(role.name, 'form .roles')
   end
 end
 
-def select_within_select2(to_select, scope)
-  tries = 3
-  begin
-    enter_name_with_select2(to_select, scope)
```

Comment:
That's what I thought too, but nope: https://github.com/opf/openproject/pull/2835/files#diff-48cc8419001b55eb4fd45894e219d88aR132